src/api/searchApi.ts

```diff
@@ -1,5 +1,4 @@
 // @ts-strict-ignore
```

Comment:
Trenger vi denne lenger?

Reply:
Yep, men jeg kan fikse akkurat denne fila for den var lett :smile: